dotcom-rendering/src/web/components/DynamicPackage.tsx

```diff
@@ -97,79 +168,448 @@ export const DynamicPackage = ({
+					headlineSize="huge"
+					imagePosition="bottom"
+					imagePositionOnMobile="bottom"
+					imageSize="large"
+					supportingContent={primary.supportingContent}
+					isDynamo={true}
 				/>
-			)}
-			<UL direction="row">
-				<LI padSides={true} percentage="75%">
-					<FrontCard
-						trail={primary}
-						containerPalette={containerPalette}
-						containerType="dynamic/package"
-						showAge={showAge}
-						headlineSize="huge"
-						imagePosition="bottom"
-						imagePositionOnMobile="bottom"
-						imageSize="large"
-						supportingContent={primary.supportingContent}
-						isDynamo={true}
-					/>
-				</LI>
-				<LI
-					showDivider={true}
-					showTopMarginWhenStacked={true}
-					percentage="25%"
-				>
-					<UL direction="column">
-						{remaining.map((card, cardIndex) => {
-							return (
-								<LI
-									key={card.url}
-									padSides={true}
-									showTopMarginWhenStacked={false}
-									padBottom={
-										// No bottom margin on the last card
-										cardIndex < remaining.length - 1
+			</LI>
+			<LI
+				showDivider={true}
+				showTopMarginWhenStacked={true}
+				percentage="25%"
+			>
+				<UL direction="column">
+					{remaining.map((card, cardIndex) => {
+						return (
+							<LI
+								key={card.url}
+								padSides={true}
+								showTopMarginWhenStacked={false}
+								padBottom={
+									// No bottom margin on the last card
+									cardIndex < remaining.length - 1
+								}
+								padBottomOnMobile={false}
+							>
+								<FrontCard
+									trail={card}
+									containerPalette={containerPalette}
+									containerType="dynamic/package"
+									showAge={showAge}
+									imageUrl={
+										// Always show the image on the first card and only
+										// on the second if there are two items in two
+										cardIndex === 0 ||
+										remaining.length === 2
+											? card.image
+											: undefined
 									}
-									padBottomOnMobile={false}
-								>
-									<FrontCard
-										trail={card}
-										containerPalette={containerPalette}
-										containerType="dynamic/package"
-										showAge={showAge}
-										imageUrl={
-											// Always show the image on the first card and only
-											// on the second if there are two items in two
-											cardIndex === 0 ||
-											remaining.length === 2
-												? card.image
-												: undefined
-										}
-										supportingContent={
-											card.supportingContent
-										}
-									/>
-								</LI>
-							);
-						})}
-					</UL>
-				</LI>
-			</UL>
-		</>
+									supportingContent={card.supportingContent}
+								/>
+							</LI>
+						);
+					})}
+				</UL>
+			</LI>
+		</UL>
 	);
 };
+
+export const DynamicPackage = ({
+	groupedTrails,
+	containerPalette,
+	showAge,
+}: Props) => {
+	let layout:
+		| 'primaryBoosted'
+		| '4orLessStandards'
+		| '5Standards'
+		| '6Standards'
+		| '7Standards'
+		| '8Standards'
+		| '9Standards';
```

Comment:
```suggestion
		| 'fourOrLessStandards'
		| 'fiveStandards'
		| 'sixStandards'
		| 'sevenStandards'
		| 'eightStandards'
		| 'nineStandards';
```

I think my brain want this is because in some languages you can't start a variable using a numeric, is that true?

Reply:
That is true - in js/ts you cannot start a variable using a numeric value, since this is a string we get away with it - but I'm happy to switch over because it does feel more readable anyway!